### PR TITLE
add customer cidr workflow

### PR DESCRIPTION
**Pull request by zhaozhanqi**

### CODE REVIEW

File: ci-operator/step-registry/cucushift/installer/custom-cidr/cucushift-installer-custom-cidr-ref.metadata.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+	"path": "cucushift/installer/custom-cidr/cucushift-installer-custom-cidr-ref.yaml",
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/custom-cidr/cucushift-installer-custom-cidr-ref.metadata.json:2]: I think this step can be shared by others, not only for QE, so suggest to move the step folder to `ci-operator/step-registry/ovn/conf/custom-cidr`.

File: ci-operator/step-registry/cucushift/installer/custom-cidr/cucushift-installer-custom-cidr-ref.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+ref:
+  as: cucushift-installer-custom-cidr
+  from_image:
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/custom-cidr/cucushift-installer-custom-cidr-ref.yaml:3]: I think this step use the same image as `ovn-conf` ref.

File: ci-operator/step-registry/cucushift/installer/custom-cidr/cucushift-installer-custom-cidr-ref.metadata.json (hunk 1 above)
Comment (zhaozhanqi) [reply ci-operator/step-registry/cucushift/installer/custom-cidr/cucushift-installer-custom-cidr-ref.metadata.json:2]: ok, good. I will move this folder.

File: ci-operator/step-registry/cucushift/installer/custom-cidr/cucushift-installer-custom-cidr-ref.yaml (hunk 2 above)
Comment (zhaozhanqi) [reply ci-operator/step-registry/cucushift/installer/custom-cidr/cucushift-installer-custom-cidr-ref.yaml]: updated, thanks

**Comment by zhaozhanqi** [comment]
@abhat Hi, Could you help take a look this PR, which is creating one custom CIDR subnet. for this bug https://bugzilla.redhat.com/show_bug.cgi?id=1980357 , base on this workflow. we will add one test case to check the alert

**Pull request merged**